### PR TITLE
Fixing the issue #390

### LINKED ISSUE #390: Title: PhoneBook Management System
### Is your feature request related to a problem? Please describe.

The phonebook management system in C is a console application without graphics.

### Describe the solution you'd like.

I will write a complete source code for storing contact in a file using C languagecontacts

### Describe alternatives you've considered.

-

### Add any other context or screenshots about the feature request here.

About: This simple mini project in C creates an external file to store the user's data permanently to perform file handling operations. Phonebook is an extremely straightforward small undertaking in C that can assist you with understanding the fundamental ideas of capacities, record taking care of, and information structure. This application will show you how to include, list, change or alter, look and erase information to/from the record.
Individual data like name, gender, father's name, contact number, postal code, email, and address are asked while including a record into the Phonebook. These records would be able to be altered, recorded, looked for, and eliminated.
We have used many functions in this project. These functions are very easy to understand as their name itself signifies their respective operations.

### PR DESCRIPTION
**Pull request by Kranthi-Guribilli**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes

### Description

(Write your answer here.)

Fixes #390 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: Desktop Application/Intermediate/C/PhoneBook_Management_System/README.md [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+		}
+	}
+	printf("Enter any key:");
+	getch();
+	system("cls");
+}
+```
+
+## Output:
+![](https://github.com/Kranthi-Guribilli/Project-Guidance/blob/main/Desktop%20Application/Intermediate/C/PhoneBook_Management_System/Output_images/Op1.png)
```
Comment (Kushal997-das) [inline Desktop Application/Intermediate/C/PhoneBook_Management_System/README.md:424]: Can you please reduce the image size ?

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Kranthi-Guribilli Can you please reduce the image size those are mentioned in your `readme.md` file .

File: Desktop Application/Intermediate/C/PhoneBook_Management_System/README.md (hunk 1 above)
Comment (Kranthi-Guribilli) [reply Desktop Application/Intermediate/C/PhoneBook_Management_System/README.md]: Yeah, has reduced the size. Please, kindly review it.

**Review by Kushal997-das — APPROVED** [review]

**Comment by Kushal997-das** [comment]
@Photon149 Check !!

**Pull request merged**